app/assets/javascripts/ckeditor/plugins/cortex_media_insert/plugin.js

```diff
@@ -0,0 +1,34 @@
+(function (global) {
+  'use strict';
+
+  global.CKEDITOR.plugins.add('cortex_media_insert', {
+    icons: 'media',
+    init: function (editor) {
+      editor.addCommand('insertMedia', {
+        exec: function (editor) {
+          window.MODALS.featured.open();
+
+          global.media_select = {};
+          global.media_select_defer = $.Deferred();
+          global.media_select_defer.promise(global.media_select);
+
+          global.media_select.done(function (media) {
```

Comment:
Oh I mean't the something like create a function like `window.initMediaSelect();` in Cortex in media_popups.js and inside cortex-plugins call it after CKEDITOR has loaded.